content/docs/setup/kubernetes/azure/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Platform setup for Azure
+description: Instructions to setup the Azure cluster for Istio.
+weight: 15
+keywords: [azure]
+---
+
+Follow these instructions to setup the Azure cluster for Istio.
+
+You can deploy a Kubernetes cluster to Azure via [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) or [ACS-Engine](https://github.com/azure/acs-engine) which fully supports Istio.
+
+#### Instructions for AKS
+
+You can create an AKS cluster via [the az cli](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) or [the Azure portal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal).
+
+For the `az` cli option, complete `az login` authentication OR use cloud shell, then run the following commands below.
+
+1. Determine the desired region name which supports AKS
+
+    {{< text bash >}}
+    $ az provider list --query "[?namespace=='Microsoft.ContainerService'].resourceTypes[] | [?resourceType=='managedClusters'].locations[]" -o tsv
+    {{< /text >}}
+
+1. Verify the supported Kubernetes versions for the desired region
+
+    Replace `my location` using the desired region value from the above step, and then execute:
+
+    {{< text bash >}}
+    $ az aks get-versions --location "my location" --query "orchestrators[].orchestratorVersion"
+    {{< /text >}}
+
+    Ensure `1.10.5` is listed or choose a different value greater than or equal to `1.9.6`.
+
+1. Create the resource group and deploy the AKS cluster
+
+    Replace `myResourceGroup` and `myAKSCluster` with desired names, `my location` using the value from step 1, `1.10.5` if not supported in the region, and then execute:
+
+    {{< text bash >}}
+    $ az group create --name myResourceGroup --location "my location"
+    $ az aks create --resource-group myResourceGroup --name myAKSCluster --node-count 3 --kubernetes-version 1.10.5 --generate-ssh-keys
+    {{< /text >}}
+
+1. Get the AKS `kubeconfig` credentials
+
+    Replace `myResourceGroup` and `myAKSCluster` with the names from the previous step and execute:
+
+    {{< text bash >}}
+    $ az aks get-credentials --resource-group myResourceGroup --name myAKSCluster
+    {{< /text >}}
+
+#### Instructions for ACS-Engine
+
+1. [Follow the instructions](https://github.com/Azure/acs-engine/blob/master/docs/acsengine.md#install) to get and install the `acs-engine` binary.
+
+1. Download the `acs-engine` API model definition that supports deploying Istio:
+
+    {{< text bash >}}
+    $ wget https://raw.githubusercontent.com/Azure/acs-engine/master/examples/service-mesh/istio.json
+    {{< /text >}}
+
+    Note: It is possible to use other api model definitions which will work with Istio.  The MutatingAdmissionWebhook and ValidatingAdmissionWebhook admission control flags and RBAC are enabled by default on 1.9 or later clusters.  See [acs-engine api model default values](https://github.com/Azure/acs-engine/blob/master/docs/clusterdefinition.md) for further information.
```

Comment:
Add the appropriate icon to differentiate the note from the steps. Respect a line length of 80 characters to allow others to edit the file with more ease. Avoid using relative pronouns such like "it" since the reference can be ambiguous and complicates translation. Don't write using passive voice. It over-complicates the content. Use the appropriate markup for the code references and to emphasize key pieces of information such as version numbers. Lastly, do not put code references within the link text since the markup does not support in-line markers within link text. 
I suggest these edits:

    {{< idea_icon >}}
    Note: Other API model definitions work with Istio. Azure enabled the
    `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook` admission
    control flags and RBAC by default on **1.9 or later** clusters. See the
    [ACS Engine API model default values docs](https://github.com/Azure/acs-engine/blob/master/docs/clusterdefinition.md)
    for further information.

Reply:
I will defer this to the engineer who added the content, I merely moved it to its own file.

Reply:
@andraxylia agree - lets keep this PR about sorting out the specific platforms.